packages/sage-assets/lib/stylesheets/themes/next/components/_search.scss

```
@@ -214,4 +214,13 @@ $-search-icon: "::before";
   .sage-search--has-text & {
     visibility: visible;
   }
+  /* stylelint-disable max-nesting-depth */
+  .sage-search & {
+    .sage-toolbar__group & {
+      &:last-child {
```

```suggestion
  .sage-toolbar__group .sage-search &:last-child {
```

If this is what you're after is it as effective? Just saying we may not need to use the nested selector per se.

---

@philschanely I am cool with that and can remove the stylelint-disable as well. 🙌🏼 - Incoming update!